src/main/java/io/github/eisop/website/EisopSiteGenerator.java

```diff
@@ -161,6 +161,9 @@ public static void main(String[] args) throws IOException {
                 e.printStackTrace();
             }
 
+            File releaseOrg = new File(String.valueOf(javadocFolder) + "/checker-javadoc/org");
+            FileUtils.copyDirectoryToDirectory(releaseOrg, javadocFolder);
```

Comment:
Isn't it enough to move the directory over? Or why do we need two copies of the javadoc?

Reply:
Yes, we could just move the directory I believe